src/Services/Air/AirErrors.js

```diff
@@ -61,15 +77,22 @@ Object.assign(AirRuntimeError, createErrorsList({
   NoValidFare: 'No valid fare for input criteria.',
   TravelersListError: 'Not all BookingTravelers present in list or wrong lookup keys provided',
 }, AirRuntimeError));
+Object.assign(AirFlightInfoRuntimeError, createErrorsList({
+  FlightNotFound: 'Flight not found',
+  AirlineNotSupported: 'Airline not supported',
+  InvalidFlightNumber: 'Invalid flight number',
+}, AirRuntimeError));
```

Comment:
Parent of the errors has to be `AirFlightInfoRuntimeError`, not `AirRuntimeError`